.github/workflows/tests.yaml

```diff
@@ -111,3 +111,17 @@ jobs:
         assignees: CAMOBAP
         update_existing: true
         filename: .github/examples-issue-template.md
+  release:
+    if: startsWith(github.ref, 'refs/tags')
```

Comment:
should only happen on master

Reply:
Done, now we check `startsWith(github.ref, 'refs/tags') && github.event_name == 'push'` and because we handle `push` only for `master` this will filter out other variants

Reply:
Maybe add a comment for this, since it's not an obvious security measure.

Reply:
I have re-checked my solution and it doesn't work, so I updated the solution, and now it looks more clear.

So if we will do a tag on the feature branch release will not happen.
BUT also after the merge, it will not happen also, because tags have no "bindings" to branches, they are just aliases to sha.

So I see several options here:
1. We can agree that we will do tags only on the master/main branch (tags on feature branches will be ignored anyway)
2. We can do the release manually (by triggering the release workflow from a button `[workflow_dispatch`](https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/) event)
3. We can implement a step as part of the release workflow that will check if the current commit is under a tag

@e271828- let me know how do you think about the options above?